apps/ledger-live-mobile/android/app/src/main/jni/OnLoad.cpp

```diff
@@ -0,0 +1,10 @@
+#include <fbjni/fbjni.h>
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)